src/components/SEO.js

```diff
@@ -8,24 +8,33 @@ export default function SEO ({
   twitterHandle,
   favicon,
   ogImage,
-  url
+  url,
+  fonts = [
+    {
+      url:
+        'https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap'
```

Comment:
Could this value be moved to `siteMetadata.js`? perhaps that file could be renamed to `site.config.js` or similar?

Reply:
Yeah I think it should be put in a file after thinking further about it

Reply:
Submitted changes in new commit below 😀